dom/src/main/scala/Dom.scala

```diff
@@ -22,6 +22,7 @@ import cats.effect.kernel.Ref
 import org.scalajs.dom
 
 import scala.scalajs.js
+import js.annotation._
 
 trait Dom[F[_]] extends js.Any
```

Comment:
I guess we should seal this.

```suggestion
sealed trait Dom[F[_]] extends js.Any
```